src/com/google/javascript/jscomp/deps/ModuleLoader.java

```diff
@@ -254,13 +278,9 @@ private String resolveJsModuleFileOrDirectory(String moduleAddress) {
 
     @Nullable
     private String resolveJsModuleDirectory(String moduleAddress) {
-      String[] extensions = {
-        MODULE_SLASH + "package.json", MODULE_SLASH + "index.js", MODULE_SLASH + "index.json"
-      };
-
       // Load as a file
-      for (int i = 0; i < extensions.length; i++) {
-        String loadAddress = locate(moduleAddress + extensions[i]);
+      for (int i = 0; i < NODE_FILES_TO_SEARCH.length; i++) {
+        String loadAddress = locate(moduleAddress + NODE_FILES_TO_SEARCH[i]);
```

Comment:
If this method is actually specific to Node.js modules, could you rename it to indicate that?